### ISSUE TITLE
German translation update (6.10)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#255]
```
Updated strings, actor conditions and itemlist for r184.

Sorry I'm on my laptop so no patch, but only the raw files.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 17 Oct 2011 at 9:32

Attachments:
- [values_de.zip](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-255/comment-0/values_de.zip)

**Comment by GoogleCodeExporter** [comment]
```
Updated 1 of the new strings.
```

Original comment by `SamuelPl...@gmail.com` on 18 Oct 2011 at 2:58

Attachments:
- [values_de.zip](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-255/comment-1/values_de.zip)

**Comment by GoogleCodeExporter** [comment]
```
Thanks. Committed in r186.
```

Original comment by `oskar.wi...@gmail.com` on 18 Oct 2011 at 6:03
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.10**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Minor update from the forum (without "questlastig")
```

Original comment by `SamuelPl...@gmail.com` on 21 Oct 2011 at 8:04

Attachments:
- [values_de.zip](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-255/comment-3/values_de.zip)

**Comment by GoogleCodeExporter** [comment]
```
Thanks. Committed in r195.
```

Original comment by `oskar.wi...@gmail.com` on 30 Oct 2011 at 1:42
- Added labels: **Milestone-0.6.11**
- Removed labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
